b2/b2.go

```diff
@@ -51,6 +51,7 @@ var (
 	uploadCutoff       = fs.SizeSuffix(200E6)
 	b2TestMode         = fs.StringP("b2-test-mode", "", "", "A flag string for X-Bz-Test-Mode header.")
 	b2Versions         = fs.BoolP("b2-versions", "", false, "Include old versions in directory listings.")
+	b2SoftDelete       = fs.BoolP("b2-soft-delete", "", true, "Hide files instead of deletion.")
```

Comment:
Can you invert the sense of this please?  I prefer the defaults for flags to be off/false and Backblaze prefer you to do the hide sort of delete by default.  So how about calling it `--b2-hard-delete`?